AsyncDisplayKit/ASCollectionNode.mm

```diff
@@ -217,6 +220,17 @@ - (_ASCollectionPendingState *)pendingState
   return _pendingState;
 }
 
+- (void)setInverted:(BOOL)inverted
+{
+  self.transform = inverted ? CATransform3DMakeScale(1, -1, 1)  : CATransform3DIdentity;
+  if ([self pendingState]) {
+    _pendingState.inverted = inverted;
+  } else {
+    ASDisplayNodeAssert([self isNodeLoaded], @"ASCollectionNode should be loaded if pendingState doesn't exist");
+    self.view.inverted = inverted;
+  }
+}
```

Comment:
You'll need to override the accessor `- (BOOL)inverted` as well (see `-delegate`). The idea is to get rid of the synthesized `_inverted` ivar, because it's not used and that's making the compiler 💣